android/app/build.gradle

```diff
@@ -36,7 +37,7 @@ android {
         minSdkVersion sdk_versions.min_sdk
         targetSdkVersion sdk_versions.target_sdk
         versionCode 1
-        versionName "1.0"
+        versionName "0.0.3"
```

Comment:
Looks like tag [v0.0.5](https://github.com/opensrp/fhircore/releases/tag/0.0.5) has already been released + tagged. 

We can update this to `v0.0.6` and tag the commit once merged and released